paddle/scripts/paddle_build.sh

```diff
@@ -575,6 +575,15 @@ EOF
             echo "Unittests with nightly labels  are only run at night"
             echo "========================================="
         fi
+
+        export GLOG_vmodule=gradient_accumulator=10,basic_engin=10 GLOG_v=5
```

Comment:
is this debug code? remove it?

Reply:
Just debug, VLOG and GLOG will remove later.